tap_facebook/__init__.py

```diff
@@ -658,6 +676,9 @@ def main_impl():
 
     CONFIG.update(args.config)
 
+    global RESULT_RETURN_LIMIT
+    RESULT_RETURN_LIMIT = CONFIG.get('result_return_limit', RESULT_RETURN_LIMIT)
```

Comment:
I'm not sure I see why this belongs here.

Reply:
Limiting the results was the first plan to fix the error, but this [stackoverflow](https://stackoverflow.com/questions/29351125/facebook-graphapi-reduce-amount-of-data-with-limit) says `limit` does not work and `since,until` is what we need.

In testing though, the tap still errors with a date window of a day and a limit of 100. I've run a test with limit=10 that seems to work

Reply:
I looked closer and follow it now.